dataprofiler/profilers/profile_builder.py

```diff
@@ -22,19 +22,20 @@
 from . import utils
 from .. import data_readers
 from .column_profile_compilers import ColumnPrimitiveTypeProfileCompiler, \
-    ColumnStatsProfileCompiler, ColumnDataLabelerCompiler
+    ColumnStatsProfileCompiler, ColumnDataLabelerCompiler, UnstructuredCompiler
 from ..labelers.data_labelers import DataLabeler
 from .helpers.report_helpers import calculate_quantiles, _prepare_report
-from .profiler_options import ProfilerOptions, StructuredOptions
+from .profiler_options import ProfilerOptions, StructuredOptions, \
+    UnstructuredOptions
 
 
-class StructuredDataProfile(object):
+class StructuredColProfiler(object):
```

Comment:
why not just `StructuredProfiler`?

Reply:
Because it exists in the class below.

Reply:
https://github.com/capitalone/DataProfiler/pull/236/files#diff-c869332fe6f1bea096dac03badb7365173fa62459962c2e9c020cb53341b1ba6R1062

Reply:
line 1062 in `profile_builder.py`